python/tvm/driver/tvmc/compiler.py

```diff
@@ -271,7 +277,7 @@ def save_module(module_path, graph, lib, params, cross=None):
         The parameters (weights) for the TVM module.
     cross : str or callable object, optional
         Function that performs the actual compilation
-
+    cross_options : sst of cross compilation options
```

Comment:
Modifying it to comply with Numpy docstring format.

```suggestion
    cross_options : str, optional
        Command line options to be passed to the cross compiler.
```